specviz/plugins/model_editor/models.py

```diff
@@ -77,7 +77,8 @@ def add_model(self, model):
             param_value.setData(parameter.value, Qt.UserRole + 1)
 
             # Store the unit information
-            param_unit = QStandardItem("{}".format(parameter.unit))
+            # param_unit = QStandardItem("{}".format(parameter.unit))
```

Comment:
Should remove this comment if it's no longer useful.

Reply:
We need the commented out lines when units are enabled. They are there because those lines are not final.